pkgs/development/libraries/LAStools/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  pname = "LAStools-${version}";
+  version = "201003";
+
+  src = fetchFromGitHub {
+    owner = "LAStools";
+    repo = "LAStools";
+    rev = "635b76b42cc4912762da31b92f875df5310e1714";
+    sha256 = "0682ca3bp51lmfp46vsjnd1bqpn05g95pf4kclvjv1y8qivkxsaq";
+  };
+
+  patches = [
+    ./drop-64-suffix.patch
+  ];
+
+  hardeningDisable = [
+    "format"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Software for rapid LiDAR processing";
+    homepage = http://lastools.org/;
+    license = stdenv.lib.licenses.unfree;
```

`stdenv.lib` is already in scope.
```suggestion
    license = licenses.unfree;
```

---

fixed in 81aecae19e628d756aeef031c7e57558b9ded030